samples/interactive-tutorials/src/main/java/product/AddFulfillmentPlaces.java

```diff
@@ -56,6 +59,13 @@ public static void addFulfillmentPlaces(String productName, String placeId)
             .setAllowMissing(true)
             .build();
 
+    // To send an out-of-order request assign the invalid AddTime here:
+    // Timestamp previousDay = Timestamp.newBuilder()
+    //        .setSeconds(Instant.now().minusSeconds(86400).getEpochSecond())
```

Comment:
Can we do minusDays?

Reply:
With Instant and protobuf timestamp only minusSeconds, so I added an instant variable, created via LocalDateTime, which look better then having minus seconds.